data/json/itemgroups/locations.json

```
@@ -588,19 +588,19 @@
       [ "trailmap", 25 ],
       [ "modern_tanner", 20 ],
       [ "pocket_survival", 20 ],
-      [ "flyer", 40 ],
-      { "item": "paper", "prob": 50, "charges-min": 1, "charges-max": 15 },
-      [ "wrapper", 35 ],
-      [ "phonebook", 55 ],
-      [ "box_small", 55 ],
-      [ "box_medium", 35 ],
-      [ "box_large", 15 ],
-      [ "one_year_old_newspaper", 45 ],
-      [ "months_old_newspaper", 50 ],
-      [ "weeks_old_newspaper", 50 ],
-      [ "one_year_old_newspaper", 45 ],
-      [ "years_old_newspaper", 40 ],
-      [ "many_years_old_newspaper", 40 ]
+      [ "flyer", 120 ],
+      { "item": "paper", "prob": 500, "charges-min": 1, "charges-max": 15 },
```

```suggestion
      { "item": "paper", "prob": 500, "charges-min": 1 },
```
No reason to cap the amount of paper, imo